intro-to-R-tidyverse/01-intro_to_base_R-live.Rmd

```diff
@@ -453,6 +453,47 @@ And view its _levels_ with the `levels()` function.
 levels(iris$Species)
 ```
 
+### Files and directories
+
+In most situations, we will be reading in data from a file and using it as a data frame. 
+To practice, we will read in a file we will be using in the next notebook as well, `gene_results_GSE44971.tsv`, in the `data` folder. 
+File paths are relative to where this notebook file (.Rmd) is saved.
+
+Here we will use a function, `read_tsv()` from the `readr` package.
+Before we are able to use the function, we have to load the package using `library()`. 
+
+```{r tidyverse}
+library(readr)
+```
+
+`file.path()` adds a file path separator (`/` on Mac and Linux operating system, which is the operating system that our RStudio Server runs on) between separate folders or directory.
+Because file path separators can differ between your computer and the computer of someone who wants to use your code, we use `file.path()` instead of typing out `"data/gene_results_GSE44971.tsv"`.
+Each _argument_ to `file.path()` is a directory or file name.
+You'll notice each argument is in quotes, we specify `data` first because it the file, `gene_results_GSE44971.tsv` is in the `data` folder. 
```

Comment:
```suggestion
You'll notice each argument is in quotes, we specify `data` first because the file, `gene_results_GSE44971.tsv` is in the `data` folder. 
```